fhireval/test_suite/crud/test_patient.py

```diff
@@ -1,85 +1,99 @@
 import pytest
 import pdb
 
+
 from fhir_walk.model.patient import Patient
 
 from fhir_walk.model import unwrap_bundle
-from fhireval.test_suite.crud import create_patient, reset_crud_test_data, example_patient
-test_id = f"{'2.5.2':<10} - CRUD Patient"
+
+from fhireval.test_suite.crud import prep_server
+import fhireval.test_suite.crud
+
+test_id = f"{'2.5.02':<10} - CRUD Patient"
 
 test_weight = 2
 
 # Just an id cache to more easily recall the record we are working with across tests
 example_patient_id = None
 
-def test_patient_create(host):
-    # Test CREATE
-    # Capture the id in case this test runs first
+
+def test_patient_create(host, prep_server):
     global example_patient_id
-    # We'll make sure that any possible left-overs from previous tests 
-    # don't make it look like our tests are passing when they did not
-    reset_crud_test_data(host)
+    # Test CREATE
+    example_patient = prep_server["CMG-Examples"]["Patient"][0]
+    response = host.post("Patient", example_patient, validate_only=False)
 
-    # Create the patient and verify that the return code marked a success
-    patient_create = create_patient(host)
-    assert patient_create['status_code'] == 201, "CREATE Success"
-    example_patient_id = patient_create['response']['id']
+    # The Patient was created above so we just need to check the return code
+    assert response["status_code"] == 201, "CREATE Success"
+    example_patient_id = response["response"]["id"]
```

Comment:
Unless we further process or print this id, this line seems redundant?

Reply:
If you mean 28, that sets the global patient_id var so that subsequent tests can use that id for their own tests.

Reply:
Ah yeah, I missed that. NVM.